model/flags.js

```diff
@@ -5,4 +5,6 @@ const VercelEnv = process.env.NEXT_PUBLIC_VERCEL_ENV || '';
 export const IsPreviewEnv = VercelEnv !== 'production';
 export const DefaultPodDomain = IsPreviewEnv
   ? 'staging.mysilio.me'
-  : 'mysilio.me';
\ No newline at end of file
+  : 'mysilio.me';
+
+export const ImportHtmlNoteBody = false
```

Comment:
I really like the direction this file is going. Seems like a super lightweight way to do flags at least until we need remote activation. For that I think we can just load the flags in this file from a POD, so glad we're getting it encapsulated early.